### ISSUE TITLE
Tween onStart signal not called with autodelay enabled

### ISSUE THREAD

**Issue by kaboompixel** [photonstorm/phaser#268]
``` javascript
// basic tween
my_tween = this.add.tween(my_sprite);

// tween with autostart=true and a delay of 1000 msecs
my_tween.to( {y: 100 }, 400, Phaser.Easing.Cubic.Out, true, 1000, false);

// should be called when tween starts
my_tween.onStart.add( function() {
    console.log('test!');
    ....
}, this);
```

the onStart is not called when the tween starts after 1 second.
However when calling my_tween.start() explicitly, it does work, but this disables the delay and starts the tween immediately.

**Comment by JeanLuc1986** [comment]
@photonstorm , I am not sure where your fix is, but it is neither in the 'dev' nor the 'master' branch.
https://github.com/photonstorm/phaser/blob/dev/src/tween/Tween.js
https://github.com/photonstorm/phaser/blob/master/src/tween/Tween.js

also when it debug the example 'tween-loop-event' onStart() is not called
http://phaser.io/examples/v2/tweens/tween-loop-event

**Comment by photonstorm** [comment]
That fix was applied almost 1.5 years ago, and worked at the time. Since then a lot has changed, so this should be opened as a new issue rather than resurrecting a very old one.
